app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <% title _("Statistics"), _('Statistics') %>
-<div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
-</div>
+
+<div id="statistics"></div>
+
+<%= javascript_tag defer: 'defer' do -%>
```

Comment:
out of curiosity - why is this inline vs a separate js file? I understand this is the glue that starts the whole process.